src/lib/CMakeLists.txt

```diff
@@ -263,6 +263,8 @@ set(
     storage/column_visitable.hpp
     storage/create_iterable_from_column.hpp
     storage/create_iterable_from_column.ipp
+    storage/defragment_reference_table.cpp
```

Comment:
sf 0,01 doesn‘t seem to be a good benchmark. With bigger tables, chunks would be bigger, too, correct?
Especially wrt your thesis, please use bigger benchmarks. You do have access to one of our servers, don't you?